packages/manager/modules/pci/src/projects/project/storages/containers/container/container.html

```diff
@@ -93,7 +93,7 @@
                 >
                     <oui-clipboard
                         data-name="containerStaticUrl"
-                        data-model="$ctrl.container.staticUrl"
+                        data-model="$ctrl.container.staticUrl || $ctrl.container.virtualHost"
```

Comment:
**question**: no issue to bind a model to a key that may not exist in the container object ?

Reply:
Well see, i don't notice any issue from my side.
It works for both case.

Also, as you know it's not a `data-ng-model` directive, and the clipboard is not editable.
So, i think no issues.